### PR TITLE
Update fauna integration documentation

### PR DESCRIPTION
**Pull request by Fedeorlandau**
Closes: ?

### What are the changes and their implications?
The base example for FaunaDB integration doesn't work out of the box. The GraphQL url depends on the database region.

## Bug Checklist

- [ ] I~ntegration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)~

## Feature Checklist

- [ ] ~Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)~
- [x] Documentation added/updated (submit PR to [blitzjs.com repo](https://github.com/blitz-js/blitzjs.com))

### CODE REVIEW

File: examples/fauna/README.md [hunk 1]
```diff
@@ -26,9 +26,12 @@ This example use the Fauna GraphQL API since it's more familiar to most people t
 
 ```
 FAUNA_SECRET=YOUR_AUTH_KEY
+FAUNA_GRAPHQL_URL=FAUNA_GRAPHQL_URL
```
Comment (beerose) [inline examples/fauna/README.md:29]: Could you fix the formatting (the numbers are not correct) and possibly mention the URL var in a separate point?

Screenshot 2021-09-29 at 13 44 33
?

Comment (Fedeorlandau) [reply examples/fauna/README.md:29]: @beerose  sure thing!

Comment (Fedeorlandau) [reply examples/fauna/README.md]: @beerose done, I've also included a link to the FaunaDB documentation to check the database region. Thanks

Comment (beerose) [reply examples/fauna/README.md]: Awesome!

**Review by beerose — APPROVED** [review]
Thanks! 🎉

**Pull request merged**